tests/common.go

```diff
@@ -358,6 +360,54 @@ func ValidateAndDestroy(contexts []*scheduler.Context, opts map[string]bool) {
 	})
 }
 
+// AddLabelsOnNode adds labels on the node
+func AddLabelsOnNode(n node.Node, labels map[string]string) error {
+	for labelKey, labelValue := range labels {
+		if err := Inst().S.AddLabelOnNode(n, labelKey, labelValue); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// ValidateStoragePools validates sizes of storage pools based on autopilot rules
+func ValidateStoragePools(contexts []*scheduler.Context, apRules []ap_api.AutopilotRule) {
```

Comment:
under the same package, create validate.go and move all Validate* methods there.

common.go is getting too big.